.travis.yml

```diff
@@ -51,7 +51,7 @@ before_install:
 
 install:
   # all mess below to get an elderly python-apt installed on elderly precise within python env -- we must build it
-  - sudo tools/ci/build_install_apt
+  - bash tools/ci/build_install_apt
```

Comment:
FWIW -- there should be no need for explicit `bash` here -- file has shebang and is executable